server/proxy/invoke_call.go

```diff
@@ -149,5 +150,12 @@ func (p *Proxy) callApp(r *incoming.Request, app *apps.App, creq apps.CallReques
 		}
 		cresp.Form = &clean
 	}
+
+	if cresp.Type != apps.CallResponseTypeError &&
+		creq.Call.Path != path.Bindings &&
```

Comment:
An app can define its own `path` for the bindings call (https://github.com/mattermost/mattermost-plugin-apps/blob/647743f10b5309af3f1b719c9b12ec19c92c1ea2/apps/manifest.go#L75-L78). That also needs to get checked.